.github/workflows/build-test.yml

```diff
@@ -10,18 +10,15 @@ concurrency:
 
 jobs:
   pytest:
-
     runs-on: ${{ matrix.os }}
     if: "github.repository == 'skops-dev/skops'"
     strategy:
-      fail-fast: false  # need to see which ones fail
+      fail-fast: false # need to see which ones fail
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python: ["3.8", "3.9", "3.10", "3.11"]
+        python: ["3.9", "3.10", "3.11"]
         # this is to make the CI run on different sklearn versions
         include:
-          - python: "3.8"
-            sklearn_version: "1.0"
```

Comment:
in the same vein as others, could you please add py312 and sklearn 1.4?

Reply:
Done! I also had to pin the macOS version in the action from @BenjaminBossan's comment: https://github.com/skops-dev/skops/pull/394#issuecomment-2082111730